.github/workflows/daily_reports.yaml

```diff
@@ -144,10 +178,10 @@ jobs:
           echo "# Daily LEVM EF Tests Run Report" >> $GITHUB_STEP_SUMMARY
           cat cmd/ef_tests/levm/levm_ef_tests_summary_github.txt >> $GITHUB_STEP_SUMMARY
 
-      - name: Post results to ethrex L2 slack channel
-        env:
-          url: ${{ secrets.ETHREX_L2_SLACK_WEBHOOK }}
-        run: sh .github/scripts/publish_levm_ef_tests_summary.sh
+      # - name: Post results to ethrex L2 slack channel
```

Comment:
why comment here?

Reply:
I think we don't want to post to this channel for now, but I might be wrong, I have to double check with @ilitteri

Reply:
Yes, we can remove these commented lines and add them later.